src/main/java/com/softserve/academy/spaced/repetition/config/SecurityConfiguration.java

```diff
@@ -94,6 +97,22 @@ public void addCorsMappings(CorsRegistry registry) {
     }
 
     @Bean
+    public CorsFilter corsFilter() {
+        UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
```

Comment:
Please create a separate pull request for security changes. It should not be committed in scope of "create tests card".